docs/diagrams/DeleteSequenceDiagram.puml

```diff
@@ -49,7 +49,7 @@ deactivate AddressBookParser
 LogicManager -> DeleteCommand : execute(m)
 activate DeleteCommand
 
-DeleteCommand -> Model : deletePerson(1)
+DeleteCommand -> Model : deletePerson(index of Person named "John Doe")
```

Comment:
I think the method `deletePerson` takes in `Person` as the parameter instead of the index